R/packages.R

```diff
@@ -286,7 +286,7 @@ install_github = function(...) remotes::install_github(...)
 
 # Remove packages not installed from CRAN
 reinstall_from_cran = function(dry_run = TRUE, skip_github = TRUE) {
-  r = paste(c('Repository', if (skip_github) 'GithubRepo'), collapse = '|')
+  r = paste(c('Repository', if (skip_github) 'GitHubRepo'), collapse = '|')
```

Comment:
https://github.com/r-lib/remotes/blob/5b7eb08f70ecc4f885f2c75330ce77f87e7dd14e/R/install-github.R#L144

```suggestion
  r = paste(c('Repository', if (skip_github) 'GithubRepo'), collapse = '|')
```